src/bindings/python/tests/test_graph/test_ops.py

```diff
@@ -117,9 +117,13 @@ def test_broadcast():
 @pytest.mark.parametrize("node", [
     Constant(Type.f32, Shape([3, 3]), list(range(9))),
     ov.constant(np.arange(9).reshape(3, 3), Type.f32),
-    ov.constant(np.arange(9).reshape(3, 3), np.float32)
+    ov.constant(np.arange(9).reshape(3, 3), np.float32),
+    ov.constant(None)
 ])
 def test_constant(node):
+    if node is None:
+        with pytest.raises(ValueError):
+            ov.constant(None)
     assert node.get_type_name() == "Constant"
     assert node.get_output_size() == 1
     assert list(node.get_output_shape(0)) == [3, 3]
```

Comment:
```suggestion
    (ov.constant, (np.arange(9).reshape(3, 3), np.float32), does_not_raise()),
    (ov.constant, [None], pytest.raises(ValueError)),
])
def test_constant(const, args, expectation):
    with expectation:
        node = const(*args)
        assert node.get_type_name() == "Constant"
        assert node.get_output_size() == 1
        assert list(node.get_output_shape(0)) == [3, 3]
```

Reply:
please, also add indentation to the line with `assert node.get_output_element_type(0) == Type.f32`, this assert need to be under `with expectation`.

and please, add `from contextlib import nullcontext as does_not_raise` in the begging of this file